code/src/components/PopularList/PopularList.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import './PopularList.css'
+
+export const PopularList = () => {
+  const [list, setList] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+    fetch('https://api.themoviedb.org/3/movie/popular?api_key=bdc909b4c34b17568b8111077d9d4d62&language=en-US&page=1')
+      .then((res) => res.json())
+      .then((data) => {
+        setList(data.results)
+      })
+      .catch((error) => {
+        console.error(console.error(error))
+      })
+      .finally(() => {
+        setLoading(false);
+      })
+  }, []);
+
+  if (loading) {
+    return (
+      <p>Loading movies...</p>
+    );
+  }
```

Comment:
Like you mentioned during the demo, you could add the setTimeOut function to check what the loading part looks like. But not important :)